server/api/rounds.py

```diff
@@ -925,6 +933,59 @@ def sample_batches(
         )
         db_session.add(sampled_batch_draw)
 
+    # Experimental feature
+    # Add extra batches on top of the original sample that will be audited, but
+    # not counted in the final risk measurement.
+    if (
+        is_enabled_sample_extra_batches_by_counting_group(election)
+        and round.round_num == 1
+    ):
+        rand = random.Random(str(election.random_seed))
```

Comment:
I'm skeptical that a second seed is needed? Provided the sample is sufficiently random, I don't think it matters whether it's truly dependent of the RLA sample. I don't think the two samples are ever likely to be combined in any meaningful way for statistical testing (in fact, you **probably shouldn't** try to do any statistical testing on the non-RLA ballots because the universe you're drawing the sample from is very much not random).

Reply:
Yeah I was thinking along similar lines to @umbernhard - since these extra sampled ballots aren't used in the RLA stats, the nature of the randomness isn't too important. But if it's easy to change the RNG, I don't have any strong opinion.